eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,98 @@
+$Language = "go"
+$packagePattern = "go.mod"
+$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+\s+(?<version>v$([AzureEngSemanticVersion]::SEMVER_REGEX))(\s+(?<releaseStatus>\(.+\))))"
+$AUTOREST_VERSION_REGEX = "^module-version:\s+(?<version>$([AzureEngSemanticVersion]::SEMVER_REGEX))"
+
+function Get-go-PackageInfoFromPackageFile ($pkg, $workingDirectory)
```

Comment:
Looks like this method can also serve as `Get-go-PackageInfoFromRepo`, sicnce go only pulls from the repo.

Reply:
Yes, Go release process has no package process and all package info can get directly from repo. But in order to reuse 'create-tag-and-git-release.ps1' script in eng sys  which use Get-go-PackageInfoFromPackageFile to get release info, I create this function by tradeoff.

Reply:
We should definitely share and reuse this logic for both the packagefile and packageinfo.

@chidozieononiwu at some point I want to eliminate FromPackageFile function completely and either use PackageInfo or use the PackageInfo.json file we are generating as part of the build.